src/AtomShell/install.jl

```diff
@@ -42,11 +42,31 @@ function install()
     end
 
     if Sys.iswindows()
-      arch = Int == Int64 ? "x64" : "ia32"
-      file = "electron-v$version-win32-$arch.zip"
-      download("https://github.com/electron/electron/releases/download/v$version/$file")
-      run(`7z x $file -oatom`)
-      rm(file)
+       arch = Int == Int64 ? "x64" : "ia32"
```

Comment:
Is there any reason you re-indented this section to 3 spaces instead of 2? Can we put it back to make comparing the diff easier?

That said, i'm not sure why the whole file has 2-space indent instead of 4... I've opened https://github.com/JunoLab/Blink.jl/pull/182 to change that, and i can just merge it after your PR is merged!  :)